### PR TITLE
Delete items api

### PR DESCRIPTION
**Pull request by jjvaris**

### CODE REVIEW

**Comment by tronza** [comment]
Is there any frontend stuff for this functionality yet?

**Comment by porsa** [comment]
I don't think there is. I haven't done any

File: lib/controllers/todolists.js [hunk 1]
```diff
@@ -87,3 +87,28 @@ exports.getTodoItem = function (req, res) {
   });
 };
 
+exports.deleteOneTodoItem = function (req, res) {
+  TodoList.findOne({"_id": new ObjectId(req.params.id)}, function(err, todoList) {
+
```
Comment (tronza) [inline lib/controllers/todolists.js:92]: Shouldn't here be some sort of user_id checking, so you could only delete stuff from your own todolist?

**Comment by porsa** [comment]
Is this functionality in master now?

**Pull request closed without merging**